File: 1-easy/problems/2951-find-the-peaks/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def findPeaks(self, mountain):
        """
        :type mountain: List[int]
        :rtype: List[int]
        """
        return [i for i in range(1, len(mountain)-1) if mountain[i-1] < mountain[i] > mountain[i+1]]

File: 1-easy/problems/2951-find-the-peaks/solution.ts
function findPeaks(mountain: any): boolean | number | string | any {
  """
  :type mountain: List[int]
  :rtype: List[int]
  """
  return [i for i in Array.from({length: 1, (mountain}, (_, i) => i).length-1) if mountain[i-1] < mountain[i] > mountain[i+1]];
}

export { findPeaks };
